cc.c

```diff
@@ -212,6 +215,7 @@ int main(int argc, char** argv)
 		}
 		else if(match(argv[i], "-V") || match(argv[i], "--version"))
 		{
+			/* Remember to update __M2C__, __M2C_PATCH__, and __M2C_PATCHLEVEL__ defines. */
 			fputs("M2-Planet v1.11.0\n", stderr);
```

Comment:
I think single source of truth is better (even if `fputs` command will be uglier). Something like
```
fputs("M2-Planet v", stderr);
fputs("__M2C__", stderr);
fputc('.', stderr);
...
```